utilities/provisioning_config_parser.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+import ansible.inventory
+
+
+def hosts_for_tag(tag):
+    hostfile = "provisioning_config"
+
+    if not os.path.isfile(hostfile):
+        print("File 'provisioning_config' not found at {}".format(os.getcwd()))
+        sys.exit(1)
```

Comment:
the perfrunner codebase had something which would check for pep8 conformance .. I think it was called flake8 or something.  I wish we had that for this repo.